assets/sharedClientFunctions.js

```diff
@@ -47,7 +48,7 @@ module.exports = {
         return "Make sure you enter all the information";
       }
       if (((ref.type === "giveaway" || ref.type === "eggcheck") && !regexpGive.test(url)) ||
```

Comment:
I think we would also need mobile versions for giveaway links

Reply:
Yeah I can add it in this one as well. I will clean up this logic like you suggested in other comment